frontend/src/features/properties/components/forms/subforms/AddressForm.tsx

```diff
@@ -7,11 +7,12 @@ import { ILookupCode } from 'actions/lookupActions';
 import { ILookupCodeState } from 'reducers/lookupCodeReducer';
```

Comment:
Thankfully this all looks simple enough to integrate with my changes.

Reply:
I attempted to essentially make one breaking change, which was the `AdministrativeArea` instead of `City` and `Municipality`. The frontend should continue working as it has been.